client/src/About.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import "./About.css" 
+import Mypic from "./img/mypic.jpg"
+function About() {
+    return (
+        <body>
+        <div class="aboutMe">
+        
+        <img class="about_pic" src={Mypic}></img>
+        <div class="about_txt">
+           <h1 class="h1_home">About Me</h1> 
```

Comment:
In react, you need to use `className` to create classes

Reply:
Hello GJ, as following you comments on my projects. The problem is i dont have much time to be doing projects as a work full time too and i work over night. and on the weekends i work during the day and night.... thats why most of my projects is not finished.
5:03
If need more explanation on how my situation is im really happy to explain, im trying my best that i can do, as my situation as a immigrant here in US is different from most people. and if i dont work i cant eat or have a place to live.  I understand that i need more dedication to this projects but my time is short as i sleep 2 hours per day...